docs/Documentation/run_RBM.md

```diff
@@ -71,15 +69,15 @@ Ndelta:         |       Usually "2", but can be larger, particularly in the case
 The control file must have the suffix, **.Control**, and the colon (:) after the descriptive characters in each line is required. <br />
 An example control file is also included in the folder **Test_case**.
 
-PS: The input/output directories need to be specified in this control file.
+Note: The input/output directories need to be specified in this control file.
 
 ### Step 4 Generate network file
-Run the Perl script, **build_input.pl**, using the control file,(**Salmon_0.5.Control**) creating the stream temperature network file (**Salmon_0.5_Network** in this example) and the ordered input file names for the routing scheme. (**Rout.Cells.init** and **Rout.Cells**). See Appendix A for a description of elements in the network file. Here again, the project name (**Salmon_0.5**) is required on the command line. The suffix, **.Control**, is appended by the Perl script:
+Run the Perl script **build_input.pl**, using the control file **Salmon_0.5.Control** creating the stream temperature network file (**Salmon_0.5_Network** in this example) and the ordered input file names for the routing scheme. (**Rout.Cells.init** and **Rout.Cells**). See Appendix A for a description of elements in the network file. Here again, the project name (**Salmon_0.5**) is required on the command line. The suffix, **.Control**, is appended by the Perl script:
```

Comment:
- Give the example command line for this step, as you did for all the other steps
- The first sentence is clunky - "Run the Perl script build_input.pl, using the control file Salmon_0.5.Control generated in the previous step as input, to create the network file required by RBM (Salmon_0.5_Network in this example) and the routing location file required for the routing scheme (Rout.Cells.init and Rout.Cells)."